Command/CreateDatabaseDoctrineCommand.php

```diff
@@ -56,7 +56,15 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $connection = $this->getDoctrineConnection($input->getOption('connection'));
+        if (empty($input->getOption('connection'))) {
```

Comment:
this does not work on PHP 5.3 and 5.4. Only variables can be passed to ``empty``